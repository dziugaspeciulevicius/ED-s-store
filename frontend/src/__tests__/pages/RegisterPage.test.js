
import React from "react";
import { shallow, mount, render } from "enzyme";
import "../../setupTests";
import RegisterPage from "../../pages/RegisterPage.component"
jest.mock("../../pages/RegisterPage.component");

describe("<RegisterPage />", () => {
    it("Renders <RegisterPage /> component correctly", () => {
      // shallow creates instance of a component
      expect(shallow(<RegisterPage />)).toMatchSnapshot();
    });
});