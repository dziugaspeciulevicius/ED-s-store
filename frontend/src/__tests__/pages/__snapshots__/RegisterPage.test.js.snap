// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<RegisterPage /> Renders <RegisterPage /> component correctly 1`] = `ShallowWrapper {}`;
